x-pack/plugins/siem/public/alerts/containers/detection_engine/signals/api.ts

```diff
@@ -54,7 +55,7 @@ export const updateSignalStatus = async ({
   query,
   status,
   signal,
-}: UpdateSignalStatusProps): Promise<unknown> =>
+}: UpdateSignalStatusProps): Promise<ReindexResponse> =>
```

Comment:
Thoughts on using the direct `elasticsearch` type here? Do we want to define our own and use that instead or is this good?

Reply:
I think we're good here. The underlying request is an `updateByQuery`, and as per the elasticsearch type defs:

``` ts
export interface UpdateDocumentByQueryResponse extends ReindexResponse {
    // DeleteDocumentByQueryResponse, UpdateDocumentByQueryResponse and ReindexResponse are identical
}
```

Reply:
To be more specific, should we use `UpdateDocumentByQueryResponse` instead, then?

Reply:
Yeah, my bad -- I can nab this one when resolving conflicts with https://github.com/elastic/kibana/pull/67731, so no need for a rebuild.